### ISSUE TITLE
Training a Single Stem

### ISSUE THREAD

**Issue by jacoblam3112** [ZFTurbo/Music-Source-Separation-Training#163]
Hi, I would like to train a model that is good at a single stem (e.g., drum). I noticed in your pretrained models section, you listed a lot of models that are good at single stem but I also noticed that these models still output other stems too. 
My question is, how is generally a single stem model trained ? I want to start with a model that is pretrained on multiple stem (e.g. HTDemucs). Then to make it good at single stem, do I disable the loss and gradient from other stems and only use the loss for the stem that I am interested in ? or is there something that I can specify in the config to achieve that ?

**Comment by deton24** [comment]
Consider visiting https://discord.gg/ZPtAU5R6rP and asking in #training
Output of other stem in single stem models if they weren't trained with 2 stem target, is just inversion to get the rest out of the output.
